Assets/MRTK/Core/Utilities/SceneContent/csc.rsp

```diff
@@ -0,0 +1 @@
+-nowarn:618
```

Comment:
I think this needs to be next to the asmdef to work.
Instead of turning this warning for the whole, kinda large core assembly, it might be fine to just wrap the offending line in 

```c#
#pragma warning disable 0618
#pragma warning restore 0618
```

Reply:
I thought the .csc only operates on the subdirectory. 

The #pragma works better though, will do that